conf/application.conf

```diff
@@ -12,27 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include "backend.conf"
-
 # This is the main configuration file for the application.
 # ~~~~~
 
+include "backend.conf"
+
 appName = individuals-savings-income-api
 
 appUrl = "http://localhost:9770"
 
-# Default http client
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
-
-# Json error handler
-play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandler"
+# These are in DES format so '2020' corresponds to the 2019-20 tax year.
+minimumPermittedTaxYear = 2020
+ukDividendsMinimumTaxYear = 2018
```

Comment:
This seems to be being used (in CreateAmendUkSavingsAccountAnnualSummaryValidator) but it seems misnamed now. Can we rename? And make sure if this and minimumPermittedTaxYear were overridden in ET/Prod configs for income-received that they have the right values for savings-income.

Reply:
- minimumPermittedTaxYear needs to be overridden in production to 2022 (meaning 2021-22)
- ukDividendsMinimumTaxYear (the one that needs renaming) doesn't look like it is overridden anywhere so the renaming can be done locally in the API.